apps/rush/CHANGELOG.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "@microsoft/rush",
   "entries": [
+    {
+      "version": "4.1.1",
+      "tag": "@microsoft/rush_v4.1.1",
+      "date": "Mon, 08 Jan 2018 20:34:30 GMT",
+      "comments": {
+        "patch": [
+          {
+            "comment": "Fix an issue with checking the shrinkwrap file when there are peer dependency version specifiers"
```

Comment:
>shrinkwrap file [](start = 55, length = 15)

PNPM lock file?

Reply:
It is still called `shrinkwrap.yaml` in pnpm